src/extension/exponent/exponentPlatform.ts

```diff
@@ -46,9 +46,14 @@ export class ExponentPlatform extends GeneralMobilePlatform {
                 },
                 (message) => {
                     return Q.Promise((resolve, reject) => {
-                        vscode.window.showInformationMessage(message)
-                            .then(password => {
-                                resolve(password || "");
+                        const okButton =  { title: "Ok" };
+                        const cancelButton =  { title: "Cancel", isCloseAffordance: true };
+                        vscode.window.showInformationMessage(message, {modal: true}, okButton, cancelButton)
```

Comment:
Is there a metric on how many users hit "Cancel" here?

Reply:
@dhanvikapila there is no metrics for now, but I could add it.